packages/electron-updater/src/providers/KeygenProvider.ts

```diff
@@ -16,7 +16,7 @@ export class KeygenProvider extends Provider<UpdateInfo> {
       ...runtimeOptions,
       isUseMultipleRangeRequest: false,
     })
-    this.baseUrl = newBaseUrl(`https://api.keygen.sh/v1/accounts/${this.configuration.account}/artifacts?product=${this.configuration.product}`)
+    this.baseUrl = newBaseUrl(`https://${this.configuration.host || "api.keygen.sh"}/v1/accounts/${this.configuration.account}/artifacts?product=${this.configuration.product}`)
```

Comment:
Minor: Let's extract this to a separate variable since it's embedding logic in a template string.
```
const host = this.configuration.host || "api.keygen.sh"
this.baseUrl = newBaseUrl(`https://${host}/v1/accounts/${this.configuration.account}/artifacts?product=${this.configuration.product}`)
```